easybuild/easyconfigs/h/HDF5/HDF5-1.8.18-gimkl-2017a.eb

```diff
@@ -0,0 +1,24 @@
+name = 'HDF5'
+version = '1.8.18'
+
+homepage = 'http://www.hdfgroup.org/HDF5/'
+description = """HDF5 is a unique technology suite that makes possible the management of
+ extremely large and complex data collections."""
+
+toolchain = {'name': 'gimkl', 'version': '2017a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = [
+    'https://support.hdfgroup.org/ftp/HDF5/current18/src/',
+    'https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-%(versions)/src/'
```

Comment:
there's a typo here (`%(versions)` should be `%(version)s`), but I've fixed this myself in #4800 since a sync with #4749 was required anyway